mesh/mesh.cpp

```diff
@@ -8613,8 +8615,8 @@ int Mesh::FindPoints(DenseMatrix &point_mat, Array<int>& elem_ids,
                for (int e = 0; e < ne; e++)
                {
                   if (els[e] == e_idx[k]) { continue; }
-                  int res = GetElementTransformation(els[e])->
-                            TransformBack(center,ips[k]);
+                  GetElementTransformation(els[e], &tr);
+                  int res = tr.TransformBack(center,ips[k], 0);
```

Comment:
I changed both uses of ``TransformBack()`` in ``mesh::FindPoints()`` to use the element center.  This reverts to the original behavior.

Reply:
Thanks!